recipes/nodejs_from_windows.rb

```diff
@@ -0,0 +1,18 @@
+#
+# Author:: Ashok Manji (ashokmanji@hotmail.com)
+# Cookbook Name:: nodejs
+# Recipe:: windows
+#
+
+arch           = node['kernel']['machine'] =~ /x86_64/ ? 'x64' : 'x86'
+version        = "v#{node['nodejs']['version']}/"
+prefix         = node['nodejs']['prefix_url'][node['nodejs']['engine']]
+filename       = "node-v#{node['nodejs']['version']}-windows-#{arch}.msi"
+nodejs_bin_url = ::URI.join(prefix, version, filename).to_s
+checksum       = node['nodejs']['binary']['checksum']["windows_#{arch}"]
```

Comment:
What about the nodejs_from_binary recipe? I based the attributes off this existing recipe...